yarn.lock

```diff
@@ -21,6 +21,11 @@
     normalize-path "^2.0.1"
     through2 "^2.0.3"
 
+"@rails/ujs@^6.0.0-alpha":
```

Comment:
Just to clarify, is the alpha necessary for the functionality we're using?  If not we should probably use a more stable version

Reply:
The versions of the UJS packages on npm are supposed to be tied to the Rails versions they originate from ([UJS is a part of Rails](https://github.com/rails/rails/tree/master/actionview/app/assets/javascripts)). The `rails-ujs` package on npm hasn't kept in sync with changes to UJS that have been pushed to RubyGems for the version of Rails it's supposed to correspond to (see https://github.com/rails/rails/pull/32404#issuecomment-436296973), in other words `"rails-ujs": "^5.2.2"` ≠ UJS ≥ 5.2.2. [This pull](https://github.com/rails/rails/pull/34905) adds a new npm package for UJS under the `@rails` scope in preparation for Rails 6 (currently alpha, but [being run in prod by Basecamp](https://weblog.rubyonrails.org/2018/12/20/timeline-for-the-release-of-Rails-6-0/)). This new package (`@rails/ujs`) has the intervening changes missing from `rails-ujs`, all of which occurred before the last stable release (5.2.2). One of the changes needed is https://github.com/rails/rails/pull/32404 which extracts the function I override [here](https://github.com/hackcambridge/hack-cambridge-website/blob/fa5bdfc3a23f866c5b8f0fd1714ec996e40f5d04/src/client/ujs.ts#L3). So in summary we need the functionality, the alpha distinction refers to Rails, not UJS, and no changes to UJS have taken place since the last Rails release. So all good I think